op-node/p2p/gossip.go

```diff
@@ -39,6 +39,8 @@ const (
 	DefaultMeshDlo   = 6  // topic stable mesh low watermark
 	DefaultMeshDhi   = 12 // topic stable mesh high watermark
 	DefaultMeshDlazy = 6  // gossip target
+	// peerScoreInspectFrequency is the frequency at which peer scores are inspected
+	peerScoreInspectFrequency = 1 * time.Minute
```

Comment:
If we only review peer scores once a minute we potentially allow a peer to cause quite a lot of issues before we actually disconnect them. https://linear.app/optimism/issue/CLI-3342/sherlock-035-memory-amplification-with-small-but-invalid-p2p-messages for example would remain an issue since the seen message TTL is about a minute anyway so they still have a full minute of sending garbage before being disconnected.

That said, evaluating peer scores is potentially expensive (it was in the java libp2p, not sure about the go version) so we may need to benchmark this when there are a significant number of known peers before making it too short.